guides/common/attributes-base.adoc

```diff
@@ -13,21 +13,49 @@
 :UpgradingDocURL: {BaseURL}Upgrading_and_Updating/{BaseFilenameURL}#
 :ReleaseNotesURL: {BaseURL}Release_notes/{BaseFilenameURL}#
 
+//Added by Marie for GA
 // Repositories and subscriptions (used both in Katello and Satellite guides)
 :SatelliteSub: Red Hat Satellite Infrastructure Subscription
-:RepoRHEL7Server: rhel-7-server-rpms
+:RepoRHEL7Server: rhel-7-server-satellite
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
 :RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
+// :RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
-// For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.10-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.10-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL8ServerAnsible: ansible-2.9-for-rhel-8-x86_64-rpms 
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.10-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-7.0-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-7.0-for-rhel-8-x86_64-rpms
+// Satellite-Tools will become Satellite-Client ??? - I haven't found this repo mentioned anywhere except for this file. So it may not be needed. The below was mentioend in 6.10 docs.
+// :RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
+// Repos relevant for Puppet - The below was mentioend for 6.10 docs.
 // Do not update the puppet4 repo versions. They must stay at 6.3.
-:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
-:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+// :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
+// :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
```

Comment:
This macro is not used anywhere in this repository. Is there any downstream-specific guide that uses it? I can imagine that now the Puppet guide is upstreamed it's no longer needed.

Reply:
OK, yes confirmed this attribute is no longer needed. I removed it. Thank you, @ekohl !